### PR TITLE
created seed tables for detector type and diagnostic threshold set testing

### PR DESCRIPTION
**Pull request by britt-allen**

### CODE REVIEW

File: transform/seeds/diag_thresh_set.csv
Comment (ian-r-rose) [inline transform/seeds/diag_thresh_set.csv:1]: This file looks like it changes from a CSV to a TSV halfway through?

Comment (britt-allen) [reply transform/seeds/diag_thresh_set.csv:1]: weird I did not catch that, thank you!

**Comment by ian-r-rose** [comment]
Do we want to think about what schema this seed data should get put in?

**Comment by ian-r-rose** [comment]
Where do these seed tables come from? Can you add documentation for them and what tables they are intended to support? (i.e., are they meant to be joined with `STATION_META`?)

**Comment by britt-allen** [comment]
Hey @kengodleskidot this is a follow up PR to the work we identified needing to be done this past Friday. Can you add your review and let me know if anything is remiss? I also need help defining `DT_NAME` and `DT_VALUE` I could not find descriptions in the docs and links provided.

**Review by ian-r-rose — APPROVED** [review]

**Pull request merged**